activesupport/lib/active_support/duration.rb

```diff
@@ -345,6 +345,49 @@ def to_s
     def to_i
       @value.to_i
     end
+    alias :to_seconds :to_i
+
+    # Returns the amount of minutes a duration covers as a float
+    #
+    #   1.day.to_minutes # => 1440.0
+    def to_minutes
```

Comment:
I think `in_minutes` (or `in_hours` etc) may read better.

Reply:
:+1: to the `in` prefix.